### PR TITLE
Add ownerreference to rendered resources ALM-402

### PR DESCRIPTION
**Pull request by ecordell**
This pull tiller into lostromos and talks to it directly to render. It configured tiller with a per-CR rendering engine that writes out ownerrefs to the rendered objects.

### CODE REVIEW

File: helmctlr/helm.go [hunk 1]
```diff
@@ -122,23 +143,66 @@ func (c Controller) installOrUpdate(r *unstructured.Unstructured) error {
...
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// tillerRendererForCR creates a ReleaseServer configured with a rendering engine that adds ownerrefs to rendered assets
+// based on the CR
+func (c Controller) tillerRendererForCR(r *unstructured.Unstructured) *tiller.ReleaseServer {
```
Comment (chancez) [inline helmctlr/helm.go:190]: Do you think we can test this method to be sure that it's correctly getting the ownerReferences of the passed in object and using those on the output objects? The `NewOwnerRefEngine` tests don't test the behavior of getting an unstructured object and getting it's ownerReferences. Possibly could just validate the created resources have it set correctly in the `TestResourceAddedHappyPathExists` test.

**Comment by chancez** [comment]
Honestly this looks pretty much fine to me. I think we could use a test case for ensuring ownerReferences are what we expect when given a CR, but other than that it all looks good.

File: helmctlr/helm.go (hunk 1 above)
Comment (ecordell) [reply helmctlr/helm.go:190]: Yes, I've validated this manually but should add a test for this

**Review by alecmerdler — APPROVED** [review]
There's a lot of changes in here but it's probably okay

File: helmctlr/helm.go (hunk 1 above)
Comment (ecordell) [reply helmctlr/helm.go:190]: Added!

**Pull request merged**